packages/core/src/utils/elementReady.ts

```diff
@@ -15,9 +17,11 @@ export const ready = function (name: string, callback?: ReadyHandler): void {
     try {
       callbackCollection.forEach((callback) => callback && callback());
     } catch (e) {
-      setTimeout(() => {
-        throw e;
-      });
+      if (isDevEnvironment) {
```

Comment:
I don't think we need this if statement. Could you double check it?

Reply:
Fixed!